### PR TITLE
Draft

### PR DESCRIPTION
**Pull request by toddmotto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,11 @@
...
+
+## Objectives
+
+- Explore the APIs and instances given to us through compile and link
+
+## Instructions
+
+Take the directive in this repo, and add a link function to it.
+
+Experiment with `compile` and `link` to the change the text in our directive to anything else.
```
Comment (ipc103) [inline README.md]: Make these instructions more explicit - tell the student exactly what the text should be changed to. 

Are there two different tasks that they could do here - one that would use `link` and one that would use `compile`?

**Comment by ipc103** [comment]
Ideally, in this lab the student would be able to practice using link and compile and then seeing what's different about them. Is there a good context where they could see the difference? 

What's the best way to test link and compile? Is there a way to test that they're using one vs the other?

**Comment by toddmotto** [comment]
We can check the textContent being changed, but can't be sure where it was changed. What would you prefer?

**Comment by ipc103** [comment]
Makes sense - yes, let's add a test to see that the textContent is being changed.

**Pull request merged**